cliconfig/config.go

```diff
@@ -178,7 +189,7 @@ func Load(configDir string) (*ConfigFile, error) {
 	}
 
 	// Can't find latest config file so check for the old one
-	confFile := filepath.Join(homedir.Get(), oldConfigfile)
+	confFile := getDefaultConfigDir(oldConfigfile)
```

Comment:
Perhaps we should keep this one as it was; wondering if we should remove this fallback completely (which version did we change to the new location?)

Note: removing the fallback to the old file should probably be a separate PR